corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -84,13 +84,15 @@
   id: "logo_android_home"
   widget: "image_uploader"
   privilege: "commcare_logo_uploader"
+  supports_linked_app: true
```

Comment:
I considered this, but decided to go this route because it felt like it involved the least change and was less prone to separate yaml files becoming out of sync with one another. Can change it around and put it in a different section if you think that would be better, but it didn't seem worth that at this point

Reply:
Nah, that makes sense to me.